src/ledctl/ledctl.c

```diff
@@ -858,6 +878,14 @@ led_status_t execute_request(struct led_ctx *ctx, struct request *req)
 
 	if (req->chosen_opt == OPT_LIST_SLOTS)
 		return list_slots(req->cntrl);
+	if (req->chosen_opt == OPT_BEST_CTRL) {
+		char device_path[PATH_MAX];
+
+		led_device_name_lookup(ctx, req->device, device_path);
+		printf("%s\n", led_cntrl_type_to_string(
```

Comment:
If the device is not supported, led_is_management_supported() will return LED_CNTRL_TYPE_UNKNOWN, and this causes "?" to be printed on the user, so, "?" doesn't say much, maybe print insteada message that requested device is not supported.